src/components/shared/ScrollTopButton.js

```diff
@@ -10,7 +10,7 @@ const ScrollTopFab = styled(Fab)`
   top: ${props => props.height - 95 + 'px'};
   right: ${props => (props.width < 1100 ? '70px' : '100px')};
   margin: 15px;
-  z-index: 9000;
+  z-index: 2;
```

Comment:
This change in Z-Index is not required.

Reply:
This is required as the scroll to top button which leads to the top of the webpage conflicts with the chat box input. See here,
![ScrollTopOverride](https://user-images.githubusercontent.com/41413622/73760535-795b1b80-4793-11ea-877d-d94035d64aef.PNG)

Reply:
The z-index puts the chat modal over the button to resolve this issue.